Update product_safety_alert_report_recalls.json

ChrisBAshton opened the pull request:
Trello: https://trello.com/c/3tKfYiAe/3348-specialist-finder-edit-request-product-safety-alerts-reports-and-recalls

---

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

ChrisBAshton reviewed (commented):
Looks like a harmless change to me. Worth calling out the removal of `topics: []` (looks like a no-op) and the removal of the `computers-and-accessories` option.

For the latter, [the docs](https://docs.publishing.service.gov.uk/repos/specialist-publisher/creating-editing-and-removing-specialist-document-types-and-finders.html) don't cover what to do when _removing_ an option. It's presumably safe to delete, but there is probably a corresponding removal PR required for Search API too, to be a good citizen? (Reindex probably not required, since we don't care about indexing on this property).

lauraghiorghisor-tw reviewed (approved):
See comments - PRs needed in search and publish apis.

ryanb-gds reviewed (approved):
Pretty sure that's all the same apart from a couple of label changes 😅

ChrisBAshton committed:
Update product_safety_alert_report_recalls.json
Trello: https://trello.com/c/3tKfYiAe/3348-specialist-finder-edit-request-product-safety-alerts-reports-and-recalls
